test/test_other.py

```diff
@@ -4477,20 +4475,14 @@ def guess_symbols_file_type(symbols_file):
         self.assertContained(UNMINIFIED_HEAP8, js)
         self.assertContained(UNMINIFIED_MIDDLE, js)
 
-  @parameterized({
-    '': [[]],
-    # bigint support is interesting to test here because it changes which
-    # binaryen tools get run, which can affect how debug info is kept around
-    'bigint': [['-sWASM_BIGINT']],
```

Comment:
Should we keep this but have `no_bigint` variant here?

Reply:
I'm not sure it's worth it. This doesn't seem like a risky area, unless I'm missing something? We tend to add coverage of non-default settings based on risk.

Reply:
sgtm